src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -82,6 +102,31 @@ pub mod tests {
         keyfile_path
     }
 
+    #[allow(dead_code)]
+    pub fn create_resources_file(name: &str) -> PathBuf {
+        let temp_dir = env::temp_dir();
+        let resources_file_path = Path::new(&temp_dir).join(name);
+
+        fs::write(
+            resources_file_path.clone(),
+            format!(
```

Comment:
You could probably just define this entire thing as a constant and skip having to format it. There are also nicer ways to make json; either with the `json!` macro or by serializing a struct.

Reply:
updated, remove the `format!` macro and use `json!` macro